bin/composite_rsr_plot.py

```diff
@@ -149,6 +149,8 @@ def get_arguments():
 
     for platform in platform_names:
         for sensor in sensors:
+            if platform in INSTRUMENTS and sensor not in INSTRUMENTS.get(platform):
+                continue
```

Comment:
This if statement seems out of place. Does this mean that it will silently not plot any sensors that it doesn't know about? Since that is coming from the user shouldn't that be an error or at least a warning?

Reply:
It is perhaps not really beautiful, but the user gives a set of platform names and instrument names, so for instance:
`--platform_name Sentinel-3B EOS-Terra --sensor olci modis`

and then here we want to avoid asking for `Terra` and `olci`.

Reply:
But you are right, I should perhaps split so first checking that Pyspectral actually supports the platform...

Reply:
The bug in issue #174 was triggered by asking for EOS-Terra and olci! ;-)
Pyspectral has and is still trying to corect what you are trying to do like here:
```
In [3]: rsr = RelativeSpectralResponse('EOS-Terra', 'olci')
Inconsistent instrument/satellite input - instrument set to modis
rsr data may not be up to date: /home/a000680/data/pyspectral/rsr_modis_EOS-Terra.h5
```

Which can be debated of course. But I am not changing that behaviour in this PR.
What I am correcting is that when asking for the channel number you would now use the method, rather than the corresponding function where you before entered with `sensor=olci` and with the (corrected) rsr now valid for Terra modis. That then gave an error. Now it is consistent and gives the correct modis band name:
```
In [4]: rsr.get_bandname_from_wavelength(3.7)
Out[4]: '20'
```

Reply:
Sounds good.